src/test/java/apoc/custom/CypherProceduresClusterTest.java

```diff
@@ -15,15 +15,10 @@
 import java.util.List;
 import java.util.Map;
 
-import static apoc.util.TestContainerUtil.cleanBuild;
-import static apoc.util.TestContainerUtil.executeGradleTasks;
-import static apoc.util.TestContainerUtil.testCall;
-import static apoc.util.TestContainerUtil.testCallInReadTransaction;
+import static apoc.util.TestContainerUtil.*;
```

Comment:
very personal opinion ^^ -> I prefer single-class imports (on the sole basis that they are grep-friendlier)

Reply:
I agree with you I just changed the notebook and forgot to change the Intellij properties, thanks for point this out!